coffee_core/src/coffee.rs

```diff
@@ -90,26 +90,26 @@ impl CoffeeManager {
     /// when coffee is configured, run an inventory to collect all the necessary information
     /// about the coffee ecosystem.
     async fn inventory(&mut self) -> Result<(), CoffeeError> {
-        self.storage
+        let _ = self
+            .storage
             .load::<CoffeeStorageInfo>(&self.config.network)
             .await
-            .and_then(|store| {
+            .map(|store| {
```

Comment:
This is a recommendation by clippy
```bash
warning: using `Result.and_then(|x| Ok(y))`, which is more succinctly expressed as `map(|x| y)`
```

Reply:
Clippy here is right. With `map` you are hiding possible errors, but in this case is not.

But I think the code can be easily solved by returning the error with `?`